go/internal/initutil/node.go

```diff
@@ -41,6 +45,12 @@ func (m *Manager) SetupLocalProtocolServerFlags(fs *flag.FlagSet) {
 	// p2p.no-ble
 }
 
+func (m *Manager) SetupProtocolAuth(fs *flag.FlagSet) {
+	m.Node.Protocol.needAuth = true
+	fs.StringVar(&m.Node.Protocol.AuthSecret, "auth.secret", "", "API Authentication Secret (base64 encoded)")
```

Comment:
flag string and comment are not clear about "what" you are talking about?

which API?